hazelcast/src/main/java/com/hazelcast/config/ReplicatedMapConfig.java

```diff
@@ -275,9 +275,11 @@ public boolean isAsyncFillup() {
      *
      * @param asyncFillup {@code true} if the replicated map is available for reads before the initial
      *                    replication is completed, {@code false} otherwise
+     * @return the current replicated map config instance
      */
-    public void setAsyncFillup(boolean asyncFillup) {
+    public ReplicatedMapConfig setAsyncFillup(boolean asyncFillup) {
```

Comment:
We can't do this because of binary compatibility: 
https://github.com/hazelcast/hazelcast/pull/11107

Reply:
We are actually allowed to do this in a minor release, since it just has to be compile time compatible (only patch releases need to be binary compatible). But Jaromir doesn't want to break this compatibility if it can be avoided.

Reply:
Ok, I'm going to revert this. Thanks for the reference.